admin/installer/_images.py

```diff
@@ -67,3 +120,269 @@ def _unserialize_packer_dict(serialized_packer_dict):
         key, value = item.split(":")
         packer_dict[key] = value
     return freeze(packer_dict)
+
+
+class PackerConfigure(PClass):
+    """
+    The attributes necessary to create a custom packer configuration file from
+    the prototype files in ``admin/installer/packer``.
+
+    :ivar build_region: The AWS region to build images in.
```

Comment:
:question: Could be a `ValueConstant` (and then the above AWS_REGION tuple could be just a ValueConstant declaration). No action required, but might be nice to catch errors earlier.